real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/EncoderNode.java

```diff
@@ -84,7 +97,8 @@ public int matchDataMinSize() {
 
 	@Override
 	public int baudRate() {
-		return 9600;
+		return 115200;
+		//return 9600;
 	}
```

Comment:
Yes, we want to get data at faster rate for the arduino.  Really frees up the arduinos time and gets us many more messages per second.